### ISSUE TITLE
Nested lists behave strangely when using spaces

### ISSUE THREAD

**Issue by karelbilek** [shurcooL/markdownfmt#54]
Another case of nested lists behaving strange. I am using spaces, not tabs.

Input:
```
- we
  - will
    - we
      - will
        - rock
          - you
            - we
              - will
                - we
                  - will
                    - rock
                      - you
```

Output - list randomly collapsed

```
-       we
        -       will
        -       we
                -       will
                -       rock
                        -       you
                        -       we
                                -       will
                                -       we
                                        -       will
                                        -       rock
                                                -       you
```

**Comment by karel-3d** [comment]
Perhaps related to https://github.com/russross/blackfriday/issues/329

**Comment by karel-3d** [comment]
Yep, it's definitely the blackfriday issue. And seeing it's a 2017 issue, it seems very hard to fix. (Trivially switching 4 to 2 in blackfriday/block.go causes other parsing issues, since Markdown is hard to parse. :((( )

**Comment by karelbilek** [comment]
#56
